### PR TITLE
Disable jit on windows

### PR DESCRIPTION
**Pull request by tienvx**
The error I am trying to fix is:

```
[Fri Sep 22 04:21:58 2023] 127.0.0.1:65468 [500]: POST /pact-change-state - Uncaught UnexpectedValueException: JIT on Windows is not currently supported in D:\a\pact-php\pact-php\vendor\phpseclib\phpseclib\phpseclib\bootstrap.php:28
```

You need this [PR](https://github.com/pact-foundation/pact-php/pull/339) to see the error clearly.

* phpseclib doesn't support [JIT on Windows](https://github.com/phpseclib/phpseclib/blob/master/phpseclib/bootstrap.php#L17), so I need to disable it.
* I tried to set env `PHPSECLIB_ALLOW_JIT` but it seems does not work

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -63,6 +63,7 @@ jobs:
           extensions: sockets, curl, zip, ffi
           php-version: ${{ matrix.php }}
           coverage: none
+          ini-values: ${{ matrix.operating-system == 'windows-latest' && 'opcache.enable=0 opcache.enable_cli=0' || '' }}
```
Comment (Lewiscowles1986) [inline .github/workflows/build.yml:66]: So this is the only change needed here. Can you separate this from the rest of the refactor and make that it's own PR?

Comment (tienvx) [reply .github/workflows/build.yml:66]: Yes, this is one line PR. The rest code is already on the [other PR](https://github.com/pact-foundation/pact-php/pull/339), which I mentioned in the PR's description.

Because of lacks of reviewers + too many PRs to manage, I usually rebase the later PR on top of the previous PR if they are somehow related, so the tests will pass + I can test them together.

I always mentioned which PRs this PR depend on, and they need to be reviewed and merged first in the description.

Comment (Lewiscowles1986) [reply .github/workflows/build.yml:66]: If this is a one-liner, I don't see how it's helping to

a) couple the refactor with it
b) not just have the one liner in the other PR

Comment (tienvx) [reply .github/workflows/build.yml:66]: Removed

**Review by Lewiscowles1986 — APPROVED** [review]

**Comment by JP-Ellis** [comment]
Is there anything else preventing this from being merged?

**Comment by Lewiscowles1986** [comment]
@JP-Ellis who is that for? I Think this is good to go, as should be most one-liners that pass tests and are not complained about, loudly, very soon 😄

**Comment by JP-Ellis** [comment]
In yesterday's Pact maintainer meetup, Tien mentioned that he was waiting on some of this PRs to be reviewed. As a result, I wanted to check if I could help. In doing so, I noticed that this PR was reviewed and approved, but still un-merged, and I was wanting to check whether there was any blockers.

**Comment by Lewiscowles1986** [comment]
Ah I'd approved as I'd commented, but it likely requires pact org members or folks with more permissions to additionally thumb. They might be busy, unless you can.

**Comment by YOU54F** [comment]
@tienvx doesn't have permission at the moment, (If I remember correctly, this was out of choice when you joined the maintainer group) - Happy to promote you when you are ready/happy 👍🏾 

I have permissions across all the repos in the foundation to merge PR's but not overly familiar with every project so appreciate others reviews from those that use each particular library! 

Thanks team - will look to get across a load of php and python PR's today :)
